homeassistant/components/switchbot/const.py

```diff
@@ -5,8 +5,9 @@
 # Config Attributes
 ATTR_BOT = "bot"
 ATTR_CURTAIN = "curtain"
+ATTR_HYDROMETER = "hydrometer"
```

Comment:
A device measuring humidity is called "hygrometer" with a `g`. A "hydrometer" measures density of liquids 😉

Reply:
Fixed it. Thanks!